packages/duckietown/include/duckietown/dtros/dtros.py

```diff
@@ -1,11 +1,27 @@
+import os
 import rospy
+from copy import copy
+
 from std_srvs.srv import SetBool, SetBoolResponse
+from duckietown_msgs.srv import \
+    NodeGetParamsList, \
+    NodeGetParamsListResponse, \
+    NodeRequestParamsUpdate, \
+    NodeRequestParamsUpdateResponse
+from duckietown_msgs.msg import NodeParameter
+from duckietown.dtros.constants import \
+    NODE_GET_PARAM_SERVICE_NAME, \
+    NODE_REQUEST_PARAM_UPDATE_SERVICE_NAME, \
+    NODE_SWITCH_SERVICE_NAME
+from .dtparam import DTParam
+from .constants import NodeHealth, NodeType
+from .diagnostics import DTROSDiagnostics
+from .utils import get_ros_handler
 
-from .dtpublisher import DTPublisher
-from .dtsubscriber import DTSubscriber
 
 class DTROS(object):
```

Comment:
How about all the things that will be used in almost every node such as NodeHealth, NodeType, DTParam are attributes of DTROS, this way for each node we will only have to import DTROS and have the whole infrastructure...